x-pack/platform/plugins/shared/fleet/server/services/epm/elasticsearch/template/__snapshots__/template.test.ts.snap

```diff
@@ -1,5 +1,1643 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`EPM template generateMappings tests loading base.yml: base.yml 1`] = `
```

Comment:
Where are these new samples used?

Reply:
I just refactored a test adding some new `describe` statements and snapshot got updated, but they're not new